tests/metrics/test_classification.py

```diff
@@ -50,7 +51,7 @@ def test_confusion_matrix(normalize, num_classes):
     target = (torch.arange(120) % 3).view(-1, 1)
     pred = target.clone()
     cm = conf_matrix(pred, target)
-    assert isinstance(cm, torch.Tensor)
+    assert isinstance(cm, torch.Tensor) and cm[torch.isnan(cm)].nelement() == 0
```

Comment:
Could you add a test that shows an example that would produce `nan`  is being converted to 0 and also test that it raises a warning (use `with pytest.warns():`)

Reply:
Writed new test for normalization at 31d22bc, but I'm not sure that this is the cleanest way to do it.